packages/angular/src/selector/selector.component.html

```diff
@@ -89,8 +85,7 @@ <h3 [textContent]="'Selected ' + (type || 'Items')"></h3>
                               [textContent]="(_.get(selectedModel, 'contentType.name') || _.get(selectedModel, 'name')) + ((_.get(selectedModel, 'status') != 1) ? ' (hidden)' : '') + ((_.get(selectedModel, 'version.published') != 1) ? ' (unpublished)' : '')">
                         </span>
                     </div>
-
-                    <h3 [innerHTML]="_.get(selectedModel, 'version.title') || _.get(selectedModel, 'version.name') || _.get(selectedModel, 'name') || (_.get(selectedModel, 'contentType.controller') === 'Content\\Module' ? _.get(selectedModel, 'version.internalIdentifier') : 'Untitled ' + _.get(selectedModel, 'contentType.name') + ' (' + _.get(selectedModel, 'id') + ')')"></h3>
+                    <h3 [innerHTML]="_.get(selectedModel, 'version.bestIdentifier')"></h3>
```

Comment:
Even though we now have the `version.bestIdetifier` reference, we should still allow the previous fallback chain to remain intact if a version isn't present for some reason.

```js
_.get(selectedModel, 'version.bestIdentifier') || _.get(selectedModel, 'name') || (_.get(selectedModel, 'contentType.controller') === 'Content\\Module' ? _.get(selectedModel, 'version.internalIdentifier') : ('Untitled ' + _.get(selectedModel, 'contentType.name') + ' (' + _.get(selectedModel, 'id') + ')' ))
```